controllers/construct/mongodbstatefulset.go

```diff
@@ -40,6 +42,7 @@ const (
 	automationConfigEnv             = "AUTOMATION_CONFIG_MAP"
 	AgentImageEnv                   = "AGENT_IMAGE"
 	MongodbImageEnv                 = "MONGODB_IMAGE"
+	MongoDBImageType                = "IMAGE_TYPE"
```

Comment:
We had a discussion to make a new env's prefixed with `MDB_`: https://docs.google.com/document/d/1jWUQePeKPycmvOf-wx1mUuppdaw6eNOZqIMqHMzwaSA/edit?disco=AAAAtnwtYX0

Reply:
Done